openmetadata-ui/src/main/resources/ui/src/utils/StringsUtils.ts

```diff
@@ -254,3 +254,21 @@ export const formatJsonString = (jsonString: string, indent = '') => {
     return jsonString;
   }
 };
+export const getUUID = () => {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(
+    /[xy]/g,
+    function (character) {
+      // Generate a random number between 0 and 15
+      const randomNumber = (Math.random() * 16) | 0;
+
+      // If the character in the UUID template is 'x', use the random number.
+      // Otherwise, use the random number ANDed with 0x3 (which gives a number between 0 and 3) ORed with 0x8
```

Comment:
Can we pull out callback function to outside and pass it's reference

```
return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(
    /[xy]/g, callbackFunc)
```

Reply:
Hello @chirag-madlani, thanks for the feedback, however, can you elaborate on what difference it will make?